Projects/Claims/Sources/Models/FlowClaimSingleItemCheckoutStepModel.swift

```diff
@@ -42,7 +42,6 @@ struct ClaimFlowMoneyModel: Codable, Equatable {
 
     func getAmountWithCurrency() -> String {
         let formatter = NumberFormatter()
-        formatter.groupingSeparator = ""
         formatter.numberStyle = .currency
         formatter.minimumFractionDigits = 0
```

Comment:
Don't forget to pass locale into this, else it will use the device one, which might not be the one we want

I think its something like this:

```suggestion
        formatter.locale = Localization.Locale.currentLocale.foundation
```

Reply:
I think thats the way to go. We should adjust to phone settings, by my opinion.
So currency decimal, group separator should be from system.
We can add it if required. 😸

Reply:
Shouldn't the locale be based on the account and not the device? Like Sam said

Reply:
Some people have en_US on their phone but the app in `sv_SE` which would entail something like `10,000 SEK` which becomes rather odd for the user, so in all other places we've just passed in the locale from Localization.Locale 🙈

Reply:
Oki  😄  thats fine if required.

Reply:
Hold on, I just realised, we already have a type `MonetaryAmount`, which currently can just be constructed from `MonetaryAmountFragment` could we maybe just make a `MoneyFragment` construct a `MonetaryAmount`, then we are 100% sure we use the same logic everywhere for this, and don't have to duplicate formatters 😄

Reply:
Used that now! :)